### ISSUE TITLE
[Bug]: Node 19 unsupported

### ISSUE THREAD

**Issue by 0rnella** [actualbudget/actual-server#151]
Moving from where I had mistakenly filed it on https://github.com/actualbudget/actual/issues/726

TLDR: Sql-lite3 version ^7.5.0 is not compatible with node@19.

We could upgrade it -- it does look like [better-sqlite3](https://github.com/WiseLibs/better-sqlite3) on the latest version, 8.1.0, installs just fine with the latest node -- but unfortunately, there is no corresponding [@types/better-sqlite3](https://www.npmjs.com/package/@types/better-sqlite3) package released yet (I have opened [a discussion](https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64626), though) and the latest one, 7.6.3, uses the incompatible version.

So the agreed solution in the meantime (discussed with @j-f1) could be to set the engines node version to be up to node@18 for easier debugging.

I say for easier debugging, because setting the engines version only gives a warning (or error, if used in conjunction with `engineStrict` if you are installing the package as a dependency) -- so this would be mostly for documentation purposes.

**Comment by j-f1** [comment]
I took a look at the release notes for the 8.x releases and I don’t see any API changes so I think it could be fine to just update the `better-sqlite3` package for now and deal with the `@types` package later.

**Comment by 0rnella** [comment]
The `@types` packages has a dependency on the `better-sqlite3` package of the same version though, so that's not an option, unfortunately.
But we can also leave this open until I hear back re: updating the types package. https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64626
